addons/im_livechat/static/src/new/core/store_service.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module */
+
+import { Store } from "@mail/new/core/store_service";
+import { _t } from "@web/core/l10n/translation";
+import { patch } from "@web/core/utils/patch";
+
+patch(Store.prototype, "im_livechat", {
+    setup(env) {
+        this._super(env);
+        this.discuss.livechat = {
+            extraClass: "o-mail-category-livechat",
+            id: "livechat",
+            name: _t("Livechat"),
+            isOpen: true,
+            canView: false,
+            canAdd: false,
+            addTitle: _t("Start a conversation"),
+            threads: [], // list of ids
+        };
```

Comment:
I don't really have a better suggestion (except defining it in the base class) but I think defining it here will prevent autocomplete in editor from working?

Reply:
Yes it will but in the other hand it does not belong to the mail module so it feels a bit weird to add it there..